packages/artifact/src/internal/config-variables.ts

```diff
@@ -6,12 +6,16 @@ export function getUploadChunkSize(): number {
   return 4 * 1024 * 1024 // 4 MB Chunks
 }
 
-export function getUploadRetryCount(): number {
-  return 3
+export function getRetryLimit(): number {
+  return 5
 }
 
-export function getRetryWaitTimeInMilliseconds(): number {
-  return 10000
+export function getRetryMultiplier(): number {
```

Comment:
Could you add comments explaining what each config option does?

It's not clear to me what exactly `RetryMultiplier` controls